.changeset/pink-panthers-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/system-access': minor
```

Comment:
They will get patched automatically with a reference to the new lib. Since their documented functionality did not change, I would not add a specific message for these modules.